src/domain/Lotto.js

```diff
@@ -0,0 +1,51 @@
+import LottoNumber from "./LottoNumber.js";
+
+class Lotto {
+  static LOTTO_NUMBER_SIZE = 6;
+  static INVALID_LOTTO_NUMBER_SIZE = "로또 번호는 6개여야 합니다.";
+  static INVALID_LOTTO_NUMBER_DUPLICATE = "로또 번호에 중복이 있습니다.";
+  static INVALID_LOTTO_NUMBER_TYPE =
+    "로또 번호는 LottoNumber 클래스여야 합니다.";
+  #lottoNumbers;
+
+  constructor(lottoNumbers) {
+    this.#validateLottoNumbers(lottoNumbers);
+    this.#lottoNumbers = [...lottoNumbers].sort(
+      (a, b) => a.getValue() - b.getValue(),
+    );
+  }
+
+  getLottoNumbers() {
+    return this.#lottoNumbers;
+  }
```

Comment:
배열의 레퍼런스를 그대로 반환하는 건 위험해 보입니다. 더욱이 private 멤버라면요..!

```suggestion
  getLottoNumbers() {
    return [...this.#lottoNumbers];
  }
```

Reply:
@Tanney-102 
감사합니다. Lotto 클래스 뿐만 아니라 LottoGame에도 같은 실수가 있어서 수정해두었습니다.